src/id3/v2/tag.rs

```diff
@@ -285,6 +288,30 @@ impl ID3v2Tag {
 
 		(None, None)
 	}
+
+	fn insert_frame(&mut self, item: TagItem) {
+		fn set_number<F: FnMut(u32)>(item: TagItem, mut setter: F) {
+			let number = item.into_value().text().map(str::parse::<u32>);
+
+			if let Some(Ok(number)) = number {
```

Comment:
Log a warning instead of silently ignoring values that couldn't be parsed successfully?

Reply:
I fixed this part.